deployment/k8s/charts/aggregates/migrationAssistantWithArgo/templates/resources/s3/createS3Bucket.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     "helm.sh/hook-delete-policy": hook-succeeded
 spec:
   ttlSecondsAfterFinished: 600
-  backoffLimit: 6
+  backoffLimit: 2
```

Comment:
Why adjust this?  3 total tries seems reasonable (is this 2 or 3 total?).
What's the failure mode overall when this step fails?